.github/workflows/e2e_tests.yml

```
@@ -37,17 +37,23 @@ jobs:
 
       - name: Run e2e tests
         working-directory: ./test
-        id: test
+        id: e2e-tests
         run: |
-          npm install
+          npm ci
           npx playwright install chromium
-          npx playwright test e2e
+          npx playwright test e2e --retries=1
+
+      - name: Upload tracing results
+        if: ${{ failure() && steps.e2e-tests.conclusion == 'failure' }}
```

```suggestion
         if: steps.e2e-tests.outcome == 'failure'
```
Will this work or not?

---

It works only when i put continue-on-error: true on steps that fails.